dllStub/ecmdDllStub.C

```diff
@@ -352,3 +353,30 @@ uint32_t dllRelatedTargets(const ecmdChipTarget & i_target, const std::string i_
   o_relatedTargets.push_back(i_target);
   return ECMD_SUCCESS;
 }
+
+uint32_t dllFapi2AttributeStringToId(std::string i_attrString, fapi2::AttributeId &o_attrId) {
```

Comment:
Shouldn't this be in some `#ifdef ECMD_FAPI2......` type block so it's only included if the user is building with fapi2 enabled?

Reply:
good point, will fix

Reply:
fixed